INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 134
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:88 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:88 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:88 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:88 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:88 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:88 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 137.0.7151.68 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 137.0.7151.68 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 137.0.7151.68
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 137.0.7151.70
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 18910 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:49489
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element/f.5734D5443DB79A2D84CA91ED9185ADB4.d.04CDDA18F309067A2AC525BE2F4C9C08.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element/f.5734D5443DB79A2D84CA91ED9185ADB4.d.04CDDA18F309067A2AC525BE2F4C9C08.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element/f.5734D5443DB79A2D84CA91ED9185ADB4.d.04CDDA18F309067A2AC525BE2F4C9C08.e.9/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: a70c9f50f05683035a68f95be37d1c2f6410fcb07856c92c8339b0ba430f2a3a
DEBUG    LoginPage:login_page.py:32 login crumb: a70c9f50f05683035a68f95be37d1c2f6410fcb07856c92c8339b0ba430f2a3a
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/window/handles HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/window/handles HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/title HTTP/1.1" 200 0
DEBUG    Header:ui_element.py:69 Click on locator ('xpath', '//a[@href="/logout"]')
DEBUG    Header:ui_element.py:28 waiting for element_to_be_clickable(('xpath', '//a[@href="/logout"]'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element/f.71D6D88EA8006DF5E7613429F328EDF1.d.7F6C7C9660F0A9E1578A9578123BD65A.e.54/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element/f.71D6D88EA8006DF5E7613429F328EDF1.d.7F6C7C9660F0A9E1578A9578123BD65A.e.54/click HTTP/1.1" 200 0
DEBUG    LoginPage:ui_element.py:28 waiting for text_to_be_present_in_element(('xpath', '//main//h1'), 'Sign in to Jenkins')
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/element/f.71D6D88EA8006DF5E7613429F328EDF1.d.7B8A420FF07CA451BFB890387615CDB2.e.11/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "POST /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/refresh HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "GET /session/1f048f6fc10b8a2d460dbfeb3bd7bb28/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49489 "DELETE /session/1f048f6fc10b8a2d460dbfeb3bd7bb28 HTTP/1.1" 200 0